internal/pkg/workspace/workspace.go

```diff
@@ -211,15 +217,16 @@ func (ws *Workspace) ListPipelines() ([]Pipeline, error) {
 			path := filepath.Join(pipelinesPath, file.Name())
 			manifest, err := ws.ReadPipelineManifest(path)
 			if err != nil {
-				log.Infof("Unable to read pipeline manifest at '%s'", path)
+				ws.logger("Unable to read pipeline manifest at '%s'", path)
```

Comment:
nit: should we include the err here:
```suggestion
				ws.logger("Unable to read pipeline manifest at '%s': %v", path, err)
```